src/syft/lib/python/slice.py

```diff
@@ -0,0 +1,204 @@
+# stdlib
+from typing import Any
+from typing import Iterable
+from typing import Optional
+from typing import Union
+
+# third party
+from google.protobuf.reflection import GeneratedProtocolMessageType
+
+# syft relative
+from ... import deserialize
+from ... import serialize
+from ...core.common import UID
+from ...core.common.serde.serializable import bind_protobuf
+from ...logger import traceback_and_raise
+from ...logger import warning
+from ...proto.lib.python.slice_pb2 import Slice as Slice_PB
+from .iterator import Iterator
+from .primitive_factory import PrimitiveFactory
+from .primitive_factory import isprimitive
+from .primitive_interface import PyPrimitive
+from .types import SyPrimitiveRet
+from .util import downcast
+from .util import upcast
```

Comment:
I see an error because these are not used 
```
src/syft/lib/python/slice.py:23:1: F401 '.util.downcast' imported but unused
from .util import downcast
^
src/syft/lib/python/slice.py:24:1: F401 '.util.upcast' imported but unused
from .util import upcast
```